package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "sanctuary-union-type",
+  "version": "0.0.0",
+  "description": "Descendant of paldepind/union-type with more descriptive error messages",
+  "main": "index.js",
+  "directories": {
+    "test": "test"
+  },
```

Comment:
Is this actually used in any way? The [documentation][1] is not particularly enlightening.


[1]: https://docs.npmjs.com/files/package.json#directories